src/content/cloud/okteto-cli.mdx

```diff
@@ -83,10 +95,17 @@ For example, for the image `registry.cloud.okteto.net/cindy/hello-world:okteto`,
 - `OKTETO_BUILD_HELLO_WORLD_REPOSITORY`: cindy/hello-world
 - `OKTETO_BUILD_HELLO_WORLD_SHA`: okteto@sha256:xxx
 
-### Built-in Tools When Deploying To Okteto Cloud
+In order to refer to endpoints of [external resources](reference/manifest.mdx/#external-object-optional) you can use the following environment variables:
+- `OKTETO_EXTERNAL_<external>_ENDPOINTS_<endpoint>_URL`: the URL of an endpoint of an external resource.
+
+In order to refer to endpoints/variables of [dependencies](reference/manifest.mdx/#dependencies-object-optional) you can use the following environment variables:
+- `OKTETO_DEPENDENCY_<dependency>_VARIABLE_<variable>`: the value of a variable of a dependency.
+- `OKTETO_DEPENDENCY_<dependency>_EXTERNAL_<external>_ENDPOINTS_<endpoint>_URL`: the URL of an endpoint of an external resource.
```

Comment:
```suggestion
- `OKTETO_DEPENDENCY_<dependency>_VARIABLE_OKTETO_EXTERNAL_<external>_ENDPOINTS_<endpoint>_URL`: the URL of an endpoint of an external resource.
```
ATM we only can consume vars from a external in a dependency from main using this notation, assuming external endpoint has been added to `OKTETO_ENV`

Reply:
I don't follow your sample, could you share a full sample?

Reply:
I think to refer the endpoint of an external resource of a dependency, the notation should be the one I was documenting

Reply:
> I don't follow your sample, could you share a full sample?

you can see the current behavior (once fixes are marged) using the following dependency: 
````yaml
dependencies:
  depend:
    repository: https://github.com/okteto/go-getting-started
    branch: generating-dynamic-envs
    variables:
      CMNAME: development
      CMDATA: hello world
    wait: true
````

This dependency has the following deploy section:
````yaml
deploy:
  - echo "OKTETO_EXTERNAL_DATABASE_ENDPOINTS_DB_URL=https://localhost:3306" >> $OKTETO_ENV
  - echo "MY_DYNAMIC_APP_ENV=hello world!" >> $OKTETO_ENV
  - kubectl apply -f k8s.yml
````
so the dependency variables available in the main pipeline will be the following:
`````env
OKTETO_DEPENDENCY_DEPEND_VARIABLE_CMDATA=hello world
OKTETO_DEPENDENCY_DEPEND_VARIABLE_OKTETO_EXTERNAL_DATABASE_ENDPOINTS_DB_URL=https://localhost:3306
OKTETO_DEPENDENCY_DEPEND_VARIABLE_CMNAME=development
OKTETO_DEPENDENCY_DEPEND_VARIABLE_MY_DYNAMIC_APP_ENV=hello world!
`````
for the moment we can only use of the variables related to the endpoints of an external using the format `OKTETO_DEPENDENCY_<dependency-name>_VARIABLE_<external-endpoint-variable-name>` if this variable has been generated and stored in `OKTETO_ENV`

Reply:
> I think to refer the endpoint of an external resource of a dependency, the notation should be the one I was documenting



> OKTETO_DEPENDENCY_<dependency>_EXTERNAL_<external>_ENDPOINTS_<endpoint>_URL

I will add your suggestion to the issue created for that feature https://github.com/okteto/okteto/issues/3599